src/injector.c

```diff
@@ -247,22 +261,41 @@ int main(int argc, char** argv)
         fprintf(stderr, "Failed to initialize on domain %s\n", domain);
         return 1;
     }
+    if (injection_thread ==0 && drakvuf_get_os_type(drakvuf) == VMI_OS_LINUX)
+    {
+        injection_thread = injection_pid;
+    }
```

Comment:
`injection_thread` doesn't seem to be used anywhere in the injector itself, I think it should be removed